ExampleMod/Items/Weapons/ExampleAnimatedSword.cs

```diff
@@ -0,0 +1,43 @@
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria;
+using Terraria.DataStructures;
+
+namespace ExampleMod.Items.Weapons
+{
+	public class ExampleAnimatedSword : ModItem
+	{
+		//PLEASE DO NOT JUST COPY/PASTE THIS CLASS AND EXPECT YOUR SWORD TO ANIMATE WHEN SWUNG. In order for that to occur, you need to insert a custom PlayerLayer. See the ModifyDrawLayers method in ExamplePlayer.
```

Comment:
Useful to add:
// See  ExampleMod/ExamplePlayer.cs for the animated PlayerLayer